src/components/vuestic-components/va-collapse/VaCollapse.demo.vue

```diff
@@ -10,7 +10,11 @@
     </VbCard>
     <VbCard title="Custom header">
       <va-collapse style="width: 400px;" customHeader>
-        <va-button slot="header" style="width: 100%">
+        <va-button
+          slot="header"
+          style="width: 100%"
+          type="collapse"
```

Comment:
Wontfix:
Please, change prop type to `is-collapse` boolean prop.  In this case, type look worse